Agents/AverageSquareMetrePriceAgent/setup.py

```diff
@@ -0,0 +1,34 @@
+from setuptools import setup, find_packages
+
+setup(
+    name='avgsqmprice',
+    version='0.0.1',
+    author='Markus Hofmeister',
+    author_email='mh807@cam.ac.uk',
+    license='MIT',
+    description="The `avgsqmprice` agent calculates the average square metre price of properties for a particular postcode and populates the result to knowledge graph as part of The World Avatar project.",
+    long_description=open('README.md').read(),
+    long_description_content_type="text/markdown",
+    packages=find_packages(exclude=("tests")),
```

Comment:
`find_namespace_packages` is now recommended for python 3.x projects

Reply:
Addressed in commit [f813e37](https://github.com/cambridge-cares/TheWorldAvatar/pull/398/commits/f813e37ede1bb95d38dd8c67ef1b03cf675734fc)